docs/use-events.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Find Events in the Catalog
+title: Event Catalog
 ---
 # About the Catalog
 
-The [Event Catalog](https://hub.lighthouse.va.gov) is the single source of truth for all events currently published on the Event Bus. The Event Catalog lives on the [Lighthouse Hub](https://hub.lighthouse.va.gov), and you will need to be on the VA network to access it.
+The [Event Catalog](https://hub.lighthouse.va.gov) is the single source of truth for all events currently published on the Event Bus. The Event Catalog lives on [CODE VA](https://hub.lighthouse.va.gov) (formerly called the Lighthouse Hub), and you will need to be on the VA network to access it.
 
 In the Event Catalog, you’ll find the following information for every event:
 
-- A brief overview of the event, including a description, domain, system of record, the topics it’s published on, the current lifecycle phase it’s in, and what other systems may be consuming it. 
+- A brief overview of the event, including a description, domain, system of record, the topics it’s published on, expected throughput, the current lifecycle phase it’s in, and what other systems may be consuming it. 
 - Data elements: A summary of what each field in an event is.
 - Schema: the overall structure of an event payload, described as an [Avro schema](https://docs.oracle.com/cd/E26161_02/html/GettingStartedGuide/avroschemas.html)
 
```

Comment:
Should we make the dashes asterisks, to be consistent across pages?

Reply:
Let's leave it for now. Marking as resolved.